src/backend/mod.rs

```diff
@@ -112,6 +112,9 @@ pub trait Backend {
     /// Get the size of the terminal screen as a [`Rect`].
     fn size(&self) -> Result<Rect, io::Error>;
 
+    /// Get the font size of the terminal screen as [`(width, height)`].
+    fn window_size(&mut self) -> Result<(Rect, (u16, u16)), io::Error>;
```

Comment:
I've been thinking about this a little - I think there's a few places within various widgets where we care more about the particular offset that a size. A good example of that `Buffer::empty()` which takes a Rect, but really only needs that to be a Size. We use this a lot for tests. So we might treat a Rect as a Point(x,y) and a Size(width, height).

It's probably out of scope for this PR to do all that however. PixelSize is still a bit ambiguous (I could see users getting confused over whether it is the size of a cell in pixels or the size of a window?)

What about:
```rust
struct WindowSize {
  size_in_rows_and_cols: Size,
  size_in_pixels: Size,
}
```

> Changing the return type of size() would also mean that this is a breaking change. Perhaps it would be better to have window_size() just return (u16, u16) as you mentioned? What if that was named `size_in_pixels()`?